### PR TITLE
verify current node version against allowed list

### PR DESCRIPTION
**Pull request by teobugslayer**
update common lib list of verified node versions

See http://teampulse.telerik.com/view#item/304396

### CODE REVIEW

File: enforce-node-version.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/**
+ * Created by totev on 30.10.2015 г..
```
Comment (rosen-vladimirov) [inline enforce-node-version.ts]: I believe we do not need this file

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by rosen-vladimirov** [comment]
:+1:  after removing the `enforce-node-version.ts` file :)

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**